forc-plugins/forc-crypto/src/keygen/new_key.rs

```diff
@@ -13,9 +13,17 @@ use libp2p_identity::{secp256k1, Keypair, PeerId};
 use serde_json::json;
 use std::ops::Deref;
 
+const ABOUT: &str = "Creates a new key for use with fuel-core";
+
+forc::cli_examples! {
+    [ Creates a new key default for block production => crypto "new-key" ]
+    [ Creates a new key for peering => crypto "new-key -k peering" ]
+    [ Creates a new key for block production => crypto "new-key -k block-production" ]
+}
+
 /// Generate a random new secret & public key in the format expected by fuel-core
 #[derive(Debug, clap::Args)]
-#[clap(author, version, about = "Creates a new key for use with fuel-core")]
+#[clap(author, version, about = ABOUT, after_long_help = help())]
```

Comment:
I'm getting the same as above with all the forc crypto examples. It shows `Fuel Labs <contact@fuel.sh>` but no examples are printed. Maybe i'm doing something wrong?

`forc crypto new-key -h`

Reply:
Agh, they show up when typing `forc crypto new-key --help` but not `forc crypto new-key -h`

Can we enable the short hand as well?